kedro/config/common.py

```diff
@@ -98,13 +103,16 @@ def _get_config_from_patterns(
     return config
 
 
-def _load_config_file(config_file: Path, ac_template: bool = False) -> Dict[str, Any]:
+def _load_config_file(
+    config_file: Path, anyconfig_args: Optional[Dict[str, Any]] = None
```

Comment:
```suggestion
    config_file: Path, anyconfig_args: Dict[str, Any]]
```
since the argument is always provided now, and the `**anyconfig_args` wouldn't actually work with the default `None` value anyway.